xdsl/utils/bitwise_casts.py

```diff
@@ -22,3 +22,11 @@ def convert_u32_to_f32(value: int) -> float:
     raw_int = ctypes.c_uint32(value)
     raw_float = ctypes.c_float.from_address(ctypes.addressof(raw_int)).value
     return raw_float
+
+
+def is_power_of_two(value: int) -> bool:
+    """
+    Return True if an integer is a power of two.
+    Powers of two have only one bit set to one
+    """
+    return (value.bit_count() - 1 == 0) and value != 0
```

Comment:
Could you please add some pytest tests for this? I'd just do `-2 to 5` as a range of tests.

Also, doesn't this give the same result?
```suggestion
def is_power_of_two(value: int) -> bool:
    """
    Return True if an integer is a power of two.
    Powers of two have only one bit set to one
    """
    return (value > 0) and (value.bit_count() == 1)
```

Reply:
That looks even prettier indeed haha

Reply:
will do